packages/server/lib/modes/run.js

```diff
@@ -1388,7 +1399,7 @@ module.exports = {
           }
 
           const runAllSpecs = ({ beforeSpecRun, afterSpecRun, runUrl, parallel }) => {
-            return this.runSpecs({
+            let runSpecs = this.runSpecs({
```

Comment:
see below, no temp var needed

```suggestion
            return this.runSpecs({
```